settings.json

```diff
@@ -0,0 +1,600 @@
+{
+    "type": "object",
+    "properties": {
+        "ALLOWED_HOSTS": {
+            "type": "array",
+            "default": []
+        },
+        "DATABASES": {
+            "type": "object",
+            "properties": {
+                "default": {
+                    "type": "object",
+                    "properties": {
+                        "NAME": {
+                            "type": "string",
+                            "default": "nautobot"
+                        },
+                        "USER": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "PASSWORD": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "HOST": {
+                            "type": "string",
+                            "default": "localhost"
+                        },
+                        "PORT": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "CONN_MAX_AGE": {
+                            "type": "integer",
+                            "default": 300
+                        },
+                        "ENGINE": {
+                            "type": "string",
+                            "default": "django.db.backends.postgresql"
```

Comment:
Wonder if we should specify the list of supported choices here, i.e `[django|django_prometheus].db.backends.[postgresql|mysql]`?

Reply:
Just to confirm, something like this?
```
"ENGINE": {
      "type": "string",
      "default": "[django|django_prometheus].db.backends.[postgresql|mysql]"
},
```

Reply:
No, something like something like https://json-schema.org/understanding-json-schema/reference/enum#enumerated-values.